langchain4j/src/main/java/dev/langchain4j/memory/chat/MessageWindowChatMemory.java

```diff
@@ -162,6 +174,17 @@ public Builder chatMemoryStore(ChatMemoryStore store) {
             return this;
         }
 
+        /**
+         * Specifies whether the system message should be added to the beginning of the message list.
+         *
+         * @param systemMessageFirst
+         * @return
+         */
+        public Builder systemMessageFirst(boolean systemMessageFirst) {
```

Comment:
```suggestion
        public Builder alwaysKeepSystemMessageFirst(boolean systemMessageFirst) {
```
Please rename it everywhere